src/shared/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,72 @@
+import React = require('react');
+import { mount } from 'enzyme';
+import getBreadcrumbs from './getBreadcrumbs';
+import { RouteWithData } from '../../routes';
+const routes: RouteWithData[] = [
```

Comment:
I have the feeling we'll end up having "real" routes vastly diverging from this set of test routes. How could we have 1 set of routes for which we override the component in order to do tests?

Reply:
We can iterate with more rigorous test cases later, right now these are simple examples that test the basic use cases